app/views/admin/shared/_status.html.haml

```diff
@@ -0,0 +1,38 @@
+-# locals: (status:, show_edited_at: true)
+
+.status__card><
+  - if status.reblog?
+    .status__prepend
+      = material_symbol('repeat')
+      = t('statuses.boosted_from_html', acct_link: admin_account_inline_link_to(status.proper.account, path: admin_account_status_path(status.proper.account.id, status.proper.id)))
+  - elsif status.reply? && status.in_reply_to_id.present?
+    .status__prepend
+      = material_symbol('reply')
+      = t('admin.statuses.replied_to_html', acct_link: admin_account_inline_link_to(status.in_reply_to_account, path: status.thread.present? ? admin_account_status_path(status.thread.account_id, status.in_reply_to_id) : nil))
+
+  = render partial: 'admin/shared/status_content', locals: { status: status.proper }
+
+  .detailed-status__meta
+    - if status.application
+      = status.application.name
+      ·
+    = link_to admin_account_status_path(status.account.id, status), class: 'detailed-status__datetime' do
+      %time.formatted{ datetime: status.created_at.iso8601, title: l(status.created_at) }= l(status.created_at)
+    - if status.edited? && show_edited_at
+      ·
+      = link_to admin_account_status_path(status.account.id, status), class: 'detailed-status__datetime' do
+        = t('statuses.edited_at_html', date: content_tag(:time, l(status.edited_at), datetime: status.edited_at.iso8601, title: l(status.edited_at), class: 'relative-formatted'))
```

Comment:
Similarly, the link should be conditional, as it does not make sense to link to the same page, or link to a page the user can't see.

Reply:
Yeah, the question then is: do I wrap it in a if/else using can? or do we want a helper "link to this thing if the user can do this on it"?